lib/src/sheet_listener_builder.dart

```diff
@@ -17,12 +16,13 @@ class SheetListenerBuilder extends StatefulWidget {
   /// Creates a widget that can be used to react to changes in the [SheetState]
   /// of a [SlidingSheet].
   const SheetListenerBuilder({
-    Key? key,
     required this.builder,
+    super.key,
     this.buildWhen,
-  }) : super(key: key);
+  });
 
   @override
+  // ignore: library_private_types_in_public_api
   _SheetListenerBuilderState createState() => _SheetListenerBuilderState();
```

Comment:
`State<SheetListenerBuilder> createState() => _SheetListenerBuilderState();` and delete ignore above